src/utils.ts

```diff
@@ -7,3 +7,16 @@ export function getNumbers(from: number, to: number): number[] {
 
```

Comment:
Now everything is written correctly, but try not to shorten the naming to unreadable names and use more understandable names ( `n` -> `num` / `number` ).